themes/src/main/resources/theme/base/login/template.ftl

```diff
@@ -125,15 +125,17 @@
 
           <#nested "form">
 
-            <#if auth?has_content && auth.showTryAnotherWayLink() && showAnotherWayIfPresent>
```

Comment:
Could you please completely remove the `showAnotherWayIfPresent` property in the _template.ftl_?

Reply:
Done!